tool/gravity/cli/status.go

```diff
@@ -380,11 +380,17 @@ func printNodeStatus(node statusapi.ClusterServer, w io.Writer) {
 		fmt.Fprintf(w, "            Status:\t%v\n", color.YellowString("offline"))
 	case statusapi.NodeHealthy:
 		fmt.Fprintf(w, "            Status:\t%v\n", color.GreenString("healthy"))
+		for _, probe := range node.WarnProbes {
+			fmt.Fprintf(w, "            [%v]\t%v\n", constants.WarnMark, color.New(color.FgYellow).SprintFunc()(probe))
```

Comment:
I'm not sure I understand the need for the extra indirection, this should do the same thing:
```go
color.New(color.FgYellow).Sprint(probe)
```